packages/linter/src/rules/IsTransformedAmp.ts

```diff
@@ -0,0 +1,17 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+export class IsTransformedAmp extends Rule {
+  run({ $ }: Context) {
+    const isTransformed = $("html[transformed='self;v=1']").length > 0;
```

Comment:
Checking for the presence of `transformed` is enough. The content might change.

Reply:
done